backend/utils/cache_service.py

```diff
@@ -1,8 +1,10 @@
 from typing import Any, Optional
 
-from account.custom_cache import CustomCache
 from django.conf import settings
 from django.core.cache import cache
+from django_redis import get_redis_connection
+
+redis_cache = get_redis_connection("default")
```

Comment:
NIT: Can this be an instance of the class?

Reply:
I think it's better to put in outside of class , There's no need to create a new instance of the connection for each instance of CacheService.

Reply:
I think it can be added as a class attribute to ensure there's only one connection for all objects of the class - but yeah we can keep it in global scope until we face an issue